cmd/osmosisd/cmd/root.go

```diff
@@ -85,6 +88,18 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+func getHomeEnvironment() string {
+	envPath := filepath.Join(osmosis.DefaultNodeHome, ".env")
+
+	// Use default node home if can't get environment
+	err := godotenv.Load(envPath)
+	if err != nil {
+		return osmosis.DefaultNodeHome
```

Comment:
I don't think we should be doing this. It will become confusing trying to figure out why this keeps giving me default when I expect something else